mod-source-record-manager-server/src/main/java/org/folio/verticle/DataImportJournalBatchConsumerVerticle.java

```diff
@@ -158,6 +159,7 @@ public List<String> getEvents() {
       DI_SRS_MARC_AUTHORITY_RECORD_NOT_MATCHED.value(),
       DI_SRS_MARC_HOLDINGS_RECORD_NOT_MATCHED.value(),
       DI_SRS_MARC_HOLDINGS_RECORD_MATCHED.value(),
+      DI_MARC_FOR_UPDATE_RECEIVED.value(),
```

Comment:
please, don't add this event because we should not subscribe to it in order not to save redundant/incorrect data to log

Reply:
Fixed. Thanks!

Reply:
thank!